### ISSUE TITLE
Crash on launch

### ISSUE THREAD

**Issue by DXXNS** [PancakeWithPotato/banana-shooter-cheat#17]
Everytime i inject the dll it gives me this error and crashes
![image](https://user-images.githubusercontent.com/108888172/219684175-0f5522c7-ba39-4398-8ecb-28a2fc39cdd8.png)

**Comment by retarded-person-chromebook** [comment]
nice codenz

**Comment by retarded-person-chromebook** [comment]
seems this is a weird problem though @PancakeWithPotato @st3ph4nnn

**Comment by DXXNS** [comment]
yeah

**Comment by DXXNS** [comment]
if you need anything just send me a dm on discord or write a comment. DXXNS#8544

**Comment by retarded-person-chromebook** [comment]
actually, can you add me on discord? Blastbrean#5236
i'll try and assist you when i'm home.

**Comment by DXXNS** [comment]
okay

**Comment by DXXNS** [comment]
thank you for your help

**Comment by DXXNS** [comment]
blasbrean can you help me now please

**Comment by retarded-person** [comment]
found issue please close.
the issue is the fact that the cheat will run into an std::runtime_error in g_Config::init.
the cheat fails to run create_directory and due to the fact there is no error handling in the current thread, the entire thread errors out.

in this case, dxxns does not have a "users" folder in his c:\ drive and therefore, create_directory errors out due to trying to create a directory in an invalid path.

**Comment by retarded-person** [comment]
@PancakeWithPotato @st3ph4nnn plz close and fix soon. ty.

**Comment by DXXNS** [comment]
i got it to work for myself.
i just chose another path in the config.cpp file
but thatn you for helping @retarded-person

**Comment by retarded-person** [comment]
yeah, but it's good to have one defined always working path for all users no matter what drive, etc.

**Comment by DXXNS** [comment]
okay

**Comment by PancakeWithPotato** [comment]
> found issue please close. the issue is the fact that the cheat will run into an std::runtime_error in g_Config::init. the cheat fails to run create_directory and due to the fact there is no error handling in the current thread, the entire thread errors out.
> 
> in this case, dxxns does not have a "users" folder in his c:\ drive and therefore, create_directory errors out due to trying to create a directory in an invalid path. (obviously because we hardcode C:\ drive, hardcode users path, and assume that everyone has a documents folder.)

I figured it was something with create_dir, as when i put couts they wouldnt print. If I replaced with mkdir, it would print, of course folders wouldnt be created. I may fix this today.
